src/com/lftechnology/java/training/alina/basic/SumOfMultiples.java

```diff
@@ -0,0 +1,31 @@
+package com.lftechnology.java.training.alina.basic;
+
+import java.util.logging.Logger;
+
+/**
+ * If we list all the natural numbers below 10 that are multiples of 3 or 5, we
+ * get 3, 5, 6 and 9. The sum of these multiples is 23. 
+ * Find the sum of all the multiples of 3 or 5 below 1000.
+ * 
+ * @author Alina Shakya<alinashakya@lftechnology.com>
+ * 
+ */
+public class SumOfMultiples {
+
+	final static Logger logger = Logger.getLogger(SumOfMultiples.class.getName());
+
+	public static void main(String[] args) {
+		try {
+			int totalSum = 0;
+			for (int i = 0; i < 1000; i++) {
+				if (i % 3 == 0 || i % 5 == 0) {
+					totalSum += i;
+				}
+			}
+			logger.info("Sum of multiples of 3 & 5 is : " + totalSum);
+		} catch (ArithmeticException ex) {
+			logger.warning("Exception : " + ex.getMessage());
+		}
+	}
```

Comment:
One request, instead of writing all the logics inside the main method, create a separate method and call it inside the main.